services/apps/alcs/src/alcs/planning-review/planning-referral/planning-referral.entity.ts

```diff
@@ -0,0 +1,49 @@
+import { AutoMap } from 'automapper-classes';
+import { Type } from 'class-transformer';
+import { Column, Entity, JoinColumn, ManyToOne, OneToOne } from 'typeorm';
+import { Base } from '../../../common/entities/base.entity';
+import { Card } from '../../card/card.entity';
+import { PlanningReview } from '../planning-review.entity';
+
+@Entity()
```

Comment:
This will need a comment based on project requirements

Reply:
Added with a migraion